sql/1.187/languages.pl

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env perl
+# recover a dev database with whatever is live on duck.co/ia/view/$repo/json enpoints
+# ./ddgc_rebuild_dev_db.pl --repo <repo to delete> --ia <ia to delete>
+# default (no options) deletes everything
+#
+use FindBin;
+use lib $FindBin::Dir . "/../../lib";
+use strict;
+use warnings;
+use DDGC;
+use Try::Tiny;
+
+my $d = DDGC->new;
+my @languages = [
+    "java",
```

Comment:
FYI, if you have single words to list like this, you can do:

```perl
qw/ java PHP HTML ... /
```

...to save typing all the quotes and commas